pkg/apis/servicecatalog/validation/instance.go

```diff
@@ -60,6 +60,10 @@ func validateInstanceSpec(spec *sc.InstanceSpec, fldPath *field.Path, create boo
 		allErrs = append(allErrs, field.Invalid(fldPath.Child("planName"), spec.PlanName, msg))
 	}
 
+	if spec.Parameters != nil && spec.AlphaSecretParameters != nil {
+		allErrs = append(allErrs, field.Invalid(fldPath.Child("alphaSecretParameters"), spec.AlphaSecretParameters, "only one of 'Parameters' and 'AlphaSecretParameters' can be set at a time"))
```

Comment:
I also mentioned this above, but I'll repeat here for completeness. There are space constraints for secrets and I think it would be appropriate to design merge rules for doing both. But I also think it's appropriate to do so outside of the scope of this PR and submit a follow-up to this PR once the design process has finished for that work. Thoughts?